Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "alsa"
-version = "0.4.2"
+version = "0.4.3"
```

Comment:
Let's skip the version changing commit for this PR - I can do a release after this PR if you like?

Reply:
That makes sense to me.